.changeset/three-dancers-share.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-query-builder': patch
+'@finos/legend-art': patch
+---
```

Comment:
Please split this into two changesets.
For legend-art don't add any description.
For legend-query-builder can you add below description:
`Add extension mechanism for query chat mode`